conf/checksums

```diff
@@ -29,6 +29,7 @@ spark:2.3.4:0241cbe73d3ad4fec1c655d839facc16e647ff9a3825f51c78d9fb270a753da4
 spark:2.3.3:a5c4a54db5163df25110534f36e13d6ae65758d45d1bc8bfcd7e27304e716a23
 spark:2.3.2:3387107155d62f04ccf6bcaf2e00a69a0de5ae5df875348d93147743c206f0a8
 spark:2.2.2:023b2fea378b3dd0fee2d5d1de6bfaf2d8349aefe7be97a9cbcf03bbacc428d7
+zookeeper:3.5.7:65c5439571cb86bc865d2a4878c83a3ed606d20cf550672e64b9bfdbbf3a8a08
```

Comment:
Can you use the SHA512 directly from the ZK download pages?

Reply:
Updated the ZK checksum to SHA512 in the new commit